python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -817,6 +817,14 @@ def cummax(self):
         """Get the column-wise cumulative maximum value in each group."""
         return self.agg("cummax")
 
+    def first(self):
+        """Get the first non-null value in each group."""
+        return self.agg("first")
+
+    def last(self):
+        """Get the last non-null value in each group."""
+        return self.agg("last")
```

Comment:
Should we add the `numeric_only` and `min_count` arguments just to be compatible with Pandas API?